README.md

```diff
@@ -1,159 +1,158 @@
 <div align="center">
-  <img src="assets/header.jpg"/><br>
-  <img src="https://github.com/tuist/tuist/workflows/Tuist/badge.svg" alt="Tuist">
-  <img src="https://img.shields.io/github/v/release/tuist/tuist?include_prereleases&style=flat-square" alt="Latest Version">
-  <img src="https://img.shields.io/github/commit-activity/w/tuist/tuist?style=flat-square" alt="Commit Activity">
-  <img src="https://img.shields.io/github/commits-since/tuist/tuist/latest?style=flat-square" alt="Latest Commits">
-  <a href="http://twitter.com/tuistio"><img src="https://img.shields.io/twitter/follow/tuistio?style=flat-square" alt="Twitter Followers"></a>
-  <a href="https://join.slack.com/t/tuistapp/shared_invite/zt-1lqw355mp-zElRwLeoZ2EQsgGEkyaFgg"><img src="https://img.shields.io/badge/slack-tuist-green.svg?logo=slack" alt="Slack Workspace"></a>
-  <img src="https://img.shields.io/opencollective/sponsors/tuistapp?style=flat-square" alt="Sponsors">
-  <img src="https://img.shields.io/opencollective/backers/tuistapp?style=flat-square" alt="Backers">
-  <img src="https://img.shields.io/github/license/tuist/tuist?style=flat-square" alt="License">
-  <img src="https://img.shields.io/badge/Powered%20by-Tuist-blue" alt="Powered by Tuist">
-  <a href="https://polar.sh/tuist"><img src="https://polar.sh/embed/seeks-funding-shield.svg?org=tuist" /></a>
-</div>
-
-<div align="center">
-  <a href="https://cal.com/team/tuist/cloud?utm_source=banner&utm_campaign=oss" target="_blank"><img alt="Book us with Cal.com" src="https://cal.com/book-with-cal-dark.svg" /></a>
+  <a href="https://tuist.io" target="_blank">
+    <picture>
+      <img src="assets/header.jpg" alt="header"/><br>
+    </picture>
+  </a>
 </div>
 
-## What's Tuist 🕺
+## 🕺 What's Tuist
 
-Tuist is a command line tool that helps you **generate**, **maintain** and **interact** with Xcode projects.
+Tuist is a command line tool that helps you **_generate_**, **_maintain_** and **_interact_** with Xcode projects.
 
 It's open source and written in Swift.
 
-## Install ⬇️
+## ⬇️ Install
 
-### Recommended: [mise](https://github.com/jdx/mise)
-
-Install [mise](https://mise.jdx.dev/getting-started.html#quickstart) and then run the following command to install Tuist:
-
-```bash
-mise install tuist
-```
```

Comment:
I'd not get rid of this. It's like buying an iPhone and being sent somewhere to read how to start using it. The first step that you need to start using Tuist, its installation, should be in the readme.

Reply:
Also before we were just redirecting to mise website on how to install mise, isn't it?